packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -146,6 +146,21 @@ class ScrollableBase extends Component {
 		 */
 		focusableScrollbar: PropTypes.bool,
 
+		/**
+		 * Specifies overscroll effects shows on which type of inputs.
+		 *
+		 * @type {Object}
+		 * @default {arrowKey: false, drag: false, pageKey: false, scrollbarButton: false, wheel: true}
```

Comment:
Could you define the default value in multiple lines?
e.g>
VideoPlayer.js
```
		 * @default {
		 *	fastForward: ['2', '4', '8', '16'],
		 *	rewind: ['-2', '-4', '-8', '-16'],
		 *	slowForward: ['1/4', '1/2'],
		 *	slowRewind: ['-1/2', '-1']
		 * }
		 * @public
		 */
		playbackRateHash: PropTypes.shape({
```

Reply:
Okay. It looks better to read. :)